satellite/console/consoleweb/consoleql/project.go

```diff
@@ -325,3 +319,16 @@ func fromMapBucketUsageCursor(args map[string]interface{}) (cursor console.Bucke
 	cursor.Search, _ = args[SearchArg].(string)
 	return
 }
+
+func fromMapProjectMembersCursor(args map[string]interface{}) (cursor console.ProjectMembersCursor) {
```

Comment:
Don't use named return parameters only for the purpose of declaring the variable (https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters)